### ISSUE TITLE
Android no longer offers to open Cyclestreets URLs in the app

### ISSUE THREAD

**Issue by oliverlockwood** [cyclestreets/android#511]
I think this is the result of an Android change a little while ago - I remember reading about how they were changing Intents etc.

Experienced it yesterday on my Pixel 4a (Android 12) when my brother sent me a link https://cycle.st/j82724336 and it just went into Chrome without giving me the option of Cyclestreets.  I had to resort to manually entering the route number instead, which was annoying.

**Comment by jezhiggins** [comment]
Fiddle fiddle fiddle. I understand the motivation for the change but this stuff is just exhausting sometimes.Relevant docs are https://developer.android.com/training/app-links/verify-site-associations#add-intent-filtersThe fix looks straightforward enough. Might have time tomorrow to pick this up. On 16 Jul 2022 12:26, Oliver Lockwood ***@***.***> wrote:
I think this is the result of an Android change a little while ago - I remember reading about how they were changing Intents etc.
Experienced it yesterday on my Pixel 4a (Android 12) when my brother sent me a link https://cycle.st/j82724336 and it just went into Chrome without giving me the option of Cyclestreets.  I had to resort to manually entering the route number instead, which was annoying.

**Comment by oliverlockwood** [comment]
Yeah, the amount of breaking changes they make to Android is pretty tiring. Cheers!

**Comment by HilaryN** [comment]
I think this is the link:
https://developer.android.com/training/app-links/
(The link above doesn't work.)
